internal/pkg/cli/svc_init.go

```diff
@@ -79,8 +79,9 @@ Deployed resources (such as your ECR repository, logs) will contain this %[1]s's
 	wkldInitDockerfilePathHelpPrompt = "Path to Dockerfile to use for building your container image."
 
 	svcInitSvcPortPrompt     = "Which %s do you want customer traffic sent to?"
-	svcInitSvcPortHelpPrompt = `The port will be used by the load balancer to route incoming traffic to this service.
-You should set this to the port which your Dockerfile uses to communicate with the internet.`
+	svcInitSvcPortHelpPrompt = `The port(s) will be used by the load balancer to route incoming traffic to this service.
+You should set this to the port(s) which your Dockerfile uses to communicate with the internet.
+You can specify multiple container ports for Load Balanced Web Services and Backend Services e.g. 3000,3001`
```

Comment:
I feel like it makes more sense to frame this prompt to "add additional path" rather than "add additional container port".